src/gwt/src/org/rstudio/studio/client/palette/ui/UserPrefBooleanPaletteEntry.java

```diff
@@ -33,18 +37,29 @@ public UserPrefBooleanPaletteEntry(BooleanValue val, UserPrefPaletteItem item)
       boolean initial = false;
 
       initial = val.getGlobalValue();
```

Comment:
super-nitpicky; could declare and assign `boolean initial` in a single line

Reply:
Done!